fvid/fvid.py

```diff
@@ -185,8 +185,7 @@ def make_video(output_filepath, image_sequence, framerate="1/5"):
             f"{FRAMES_DIR}encoded_frames*.png",
             pattern_type="glob",
             framerate=framerate,
-        ).output(outputfile, vcodec="libx264rgb").run(quiet=True)
-
+        ).output(outputfile, r=30, vcodec="libx264rgb").run(quiet=True)
```

Comment:
I believe you are on the right track. This works for the default frame rate of "1/5", but uit would be even better if you can figure out how to make it work for all framerates.

Great work so far, I would love to get this bug fixed ion Mac, and I think this would alos help on other system. I saw something similar of Linux and I think it has to do with the same thing.